remote-api/src/main/java/org/alfresco/rest/api/Rules.java

```diff
@@ -57,4 +59,14 @@
      * @return {@link Rule} definition
      */
     Rule getRuleById(String folderNodeId, String ruleSetId, String ruleId);
+
+    /**
+     * Create new rules (and potentially a rule set if "_default_" is supplied).
+     *
+     * @param folderNodeId The node id of a folder.
+     * @param ruleSetId The id of a rule set (or "_default_" to use/create the default rule set for the folder).
+     * @param rule The definition of the rule.
+     * @return The newly created rules.
+     */
+    List<Rule> saveRules(String folderNodeId, String ruleSetId, List<Rule> rule);
```

Comment:
Minor: rename "saveRules" as "createRules" ?

Reply:
Yes - good suggestion. This will now match with the other APIs classes.